src/bindings/python/tests/test_runtime/test_ovdict.py

```diff
@@ -0,0 +1,249 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
should we also tests for InferRequest which show that tensors can be accessed using new approach?

Reply:
All of these "tensor-getters" are present since 2022.1 release.
Some tests:
https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/tests/test_runtime/test_infer_request.py#L125
https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/tests/test_runtime/test_infer_request.py#L202
There are no changes to `InferRequest` itself in this PR. Only affected part is the return value of specific infer functions. Thus new tests would not cover anything more.

@ilya-lavrenov Did I understand you correctly?